include/mm/pmm.h

```diff
@@ -63,6 +63,9 @@ static inline bool paddr_invalid(paddr_t pa) {
 static inline bool mfn_invalid(mfn_t mfn) { return paddr_invalid(mfn_to_paddr(mfn)); }
 
 static inline frame_t *get_free_frame(void) { return get_free_frames(PAGE_ORDER_4K); }
+static inline void put_free_frame(mfn_t mfn) {
```

Comment:
put_free_frame and put_free_frames doing same work?

Reply:
Yes, but `put_free_frame()` is a wrapper for 4K frames.

Reply:
Can we rename it as such? It might get confusing for any new developer.

Reply:
I would keep it as-is for this PR. Probably makes sense to add another change on top with all the relevant helpers (properly named).

Reply:
alright, lgtm then.